### PR TITLE
Add Gojek Tech

### PR DESCRIPTION
**Pull request by anik3tra0**

### CODE REVIEW

**Comment by kamleshchandnani** [comment]
Hey @anik3tra0,
Thank you so much for your contribution 🎉

File: content/companies/06-gojek-tech.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Gojek Tech
+excerpt: Know more about Gojek Tech's interview process ➡️
+index: 6
```
Comment (kamleshchandnani) [inline content/companies/06-gojek-tech.md]: you can remove this `index` key. This was old design now I'm taking the filename as the index name so less hassle on the developer's end

File: content/companies/06-gojek-tech.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+## Interview Process Summary
+|                                    |              |
+| :--------------------------------: | :-----------:|
+|             **Rounds**             |  ~6/7        |
+|      **Scripted Questions?**       |  No          |
+| **Typical DS and Algo Questions?** |  Sometimes   |
+|          **Live Coding?**          |  Yes         |
+
+## Rounds Description
+0. **Assignment:** To begin with candidates receive a take home coding assignment, which is a real life problem and are given 2 full days to solve it. Once the candidate submits the assignment, it undergoes a basic code review and upon successful checks, the candidate is then invited to the next round. This is either online or on premise depending upon candidates feasibility.
```
Comment (kamleshchandnani) [inline content/companies/06-gojek-tech.md]: List is a numbered list and the index starts from 1. so 1, 2, 3 … and so on

**Comment by kamleshchandnani** [comment]
In addition to the info above can you add some more info in the comments here? I'll be updating the format to include information of the reviewer.
Information needed:
1. Name
2. Twitter Handle
3. Role/Level (description of what role you had applied/adding reviews for)

**Comment by kamleshchandnani** [comment]
Hey @anik3tra0,
Are you considering this and taking it up?

File: content/companies/06-gojek-tech.md [hunk 3]
```diff
@@ -16,14 +15,20 @@ index: 6
...
 - **Scripted Questions** - Describes if the company asks questions which are kind of direct bookish questions or can be easily found on internet infact on the first result page on google 😛. Example: Write a function to print first 100 fibonnaci numbers, How does event loop works, write a polyfil for X etc.
-- **DS and Algo** - Describes if the company asks DS and algo questions which which are kind of direct bookish questions about data structure and algorithms instead of a real life problem.
\ No newline at end of file
+- **DS and Algo** - Describes if the company asks DS and algo questions which which are kind of direct bookish questions about data structure and algorithms instead of a real life problem.
+
+
+## Reporter Info
+- **Name:** Aniket Rao
+- **Twitter:** [@aniket_rao](https://twitter.com/aniket_rao)
+- **Role/Level:** Product Engineer
```
Comment (kamleshchandnani) [inline content/companies/06-gojek-tech.md]: Is this your current role? Or the role/level this process belongs to?

Comment (anik3tra0) [reply content/companies/06-gojek-tech.md]: Is my current role and also the most frequently hired role and also the current process.

**Pull request merged**